src/TreeMenu/walk.tsx

```diff
@@ -80,23 +87,23 @@ const walk = ({ data, ...props }: WalkProps): Item[] => {
     : handleObject(validatedData);
 };
 
-const matchSearch = (label: string, searchTerm: string) => {
+const defaultMatchSearch = ({label, searchTerm}: MatchSearchFunctionProps) => {
```

Comment:
There should be a space before `label` and after `searchTerm`. Did you configure prettier on your IDE?